### PR TITLE
Translate "loop" to Ukrainian

### PR DESCRIPTION
**Pull request by Herobread**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Oleksii Nazarenko

## Language: 

- Ukrainian

## Terms defined:

- loop

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -6809,6 +6809,10 @@
     def: >
       También conocido como bucle. Estructura que ejecuta un bloque de código repetidamente hasta
       que se cumple una condición de salida o fin.
+  uk:
+    term: "цикл"
+    def: >
+      Структура, яка повторно виконує ділянку коду, доки не буде виконана умова виходу.
```
[the remark this answers is not in the document]
Comment (Herobread) [reply glossary.yml]: так, краще і коректніше звучить

**Review by olexandr-konovalov — APPROVED** [review]

File: glossary.yml [hunk 2]
```diff
@@ -6809,6 +6809,10 @@
     def: >
       También conocido como bucle. Estructura que ejecuta un bloque de código repetidamente hasta
       que se cumple una condición de salida o fin.
+  uk:
+    term: "цикл"
+    def: >
+      Структура, яка повторно виконує фрагмент коду, доки не буде виконана умова виходу.
```
Comment (olexandr-konovalov) [inline glossary.yml:6815]: Давайте напишемо "Структура у програмі, яка ...

Comment (Herobread) [reply glossary.yml]: Давайте

**Comment by olexandr-konovalov** [comment]
@sofiiamatios @VeronikaShevc @Dod900ls1 - could one or more of you review this please

**Review by Dod900ls1 — APPROVED** [review]
LGTM!

**Review by olexandr-konovalov — APPROVED** [review]
@elletjies please merge

**Pull request merged**